corda/corda-runtime-os
# CORE-9865 Creation of InterOpFacade Processor  - send a seed message

Send a seed message allows to observe interaction between InteropProcessor and LinkManager:
1)  Enabled invocation of `seedMessage` to publish seed message
2) Added temporary logic for seed messages only, to process the first 10 messages as more is not required and they were where over-flooding the system,
for this check needed 2 more temporary private methods to parse message id and increment the part with number
3) Temporary log info statements to LinkManager processors for unauthorized message.
4) rename `appMessage` to `record` in `InteropProcessor` loop as the latter variable contains ApplicationMessage
5) Several refactorings in our registration service, this is to remove accidentally constants used to create other constants, shorten names (`_NAME`postfix not needed), more compact way of creating a list using map function.
For follow-up code in the next PR - move logic from public methods `createDummyMemberInfo` and `createDummyHostedIdentity` into private function with arguments, as the logic will be used more times to create various setup for other seed message scenarios (no matching group policy etc).

How to test:
Download any logs from Jenkins from e2e test, go to any p2p link manager :
```
cd run-b83b94c5-b13f-4510-a93c-ffceb790a818/corda-alice-p2p-link-manager-worker-86787595c4-j5f8t
 cat logs.txt| grep -e OutboundMessageProcessor | grep Alice |  cut -d] -f3-
```
Sample line should a message received by `LinkManager` from p2p.out and send back  to p2p.in. There is 11 such messages send from `Alice Alter Ego` to `Alice` back and forth, you can also see the logs from `InteropProcessor` from relevant `FlowWorker`node (use e.g.:
```
cat logs.txt | grep interop | cut -d] -f3-
```
